modules/theora/video_stream_theora.cpp

```diff
@@ -451,7 +451,6 @@ void VideoStreamPlaybackTheora::set_file(const String &p_file) {
 					ti.frame_width, ti.frame_height, ti.pic_x, ti.pic_y);
 		th_decode_ctl(td, TH_DECCTL_GET_PPLEVEL_MAX, &pp_level_max,
 				sizeof(pp_level_max));
-		pp_level = pp_level_max;
 		pp_level = 0;
```

Comment:
No I guess it's fine, it was likely a bug fix (if not, git blame will tell the next interested guy the little we know :p).